### PR TITLE
(BKR-1507) Specify vcenter_instance for init

### PR DESCRIPTION
**Pull request by jcoconnor**
Allow alternative vcenter instance credentials to be used for the
connection to allow for acceptance testing away from the default
vcenter.

### CODE REVIEW

**Comment by jcoconnor** [comment]
Initial WIP PR associated with https://github.com/puppetlabs/beaker-vmware/pull/8

File: lib/beaker/hypervisor/vcloud.rb [hunk 1]
```diff
@@ -22,10 +22,14 @@ def initialize(vcloud_hosts, options)
       @logger = options[:logger]
       @hosts = vcloud_hosts
 
+      # Allow vcenter instance to be specified so that specific credentials can be used
+      # rather than the top-level credentials in the .fog file
+      vcenter_instance = options[:vcenter_instance] || ""
```
Comment (ekinanp) [inline lib/beaker/hypervisor/vcloud.rb]: Should be `vcenter_instances`.

**Comment by jcoconnor** [comment]
I've tested this using a test ```.fog``` file against ```vcenter-prod1``` and it connects correctly:
```
bundle exec beaker  -h ../hosts.yaml provision
Beaker::Hypervisor, found some vcloud boxes to create
WARNING: Nokogiri was built against LibXML version 2.9.7, but has dynamically loaded 2.9.4
Connecting to vSphere at vcenter-prod1.ops.puppetlabs.net with credentials for sre-template-creator@puppet.com
Deploying bo0bygefsxxt45t (windows10pro-64-1) to packer/acceptancehosts from template 'win-10-pro-x86_64-20180627_DEV'
Spent 10.09 seconds deploying VMs
Booting bo0bygefsxxt45t (windows10pro-64-1) and waiting for it to register with vSphere
```

**Comment by jcoconnor** [comment]
Ok - thats the tests fixed up.
Can we get this merged now please ??????

And probably need a ```beaker-vcloud``` release so we can get 
[IMAGES-877](https://tickets.puppetlabs.com/browse/IMAGES-877) unblocked.

**Pull request merged**